syntax/python.vim

```diff
@@ -75,6 +75,7 @@ endif
     syn keyword pythonStatement yield
     syn keyword pythonLambdaExpr lambda
     syn keyword pythonStatement with as
+    syn keyword pythonStatement async await
```

Comment:
I notice https://github.com/klen/python-mode/pull/627/files is a bit more elaborate than this.  What's the difference ?